ironfish-cli/src/commands/wallet/multisig/participants/index.ts

```diff
@@ -5,7 +5,7 @@ import { Flags } from '@oclif/core'
 import { IronfishCommand } from '../../../../command'
 import { RemoteFlags } from '../../../../flags'
 
-export class MultisigAccountIdentities extends IronfishCommand {
+export class MultisigAccountParticipants extends IronfishCommand {
   static description = `List the identities for a multisig account`
```

Comment:
Should we document this as 'participant identities'?

Reply:
Updated